src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -72,20 +73,25 @@ namespace creators {
  *      object, this layer transitions from a sphere to a cube.
  * - 3: The blocks that surround each cube with a half-cube. At this layer, the
  *      two compact objects are enclosed in a single cube-shaped grid.
- * - 4: The blocks that form the outer sphere. This layer transitions back to
- *      spherical and can extend to large radial distances from the compact
- *      objects.
+ * - 4: The 10 blocks that form the first outer shell. This layer transitions
+ *      back to spherical. The gridpoints are distributed linearly with respect
+ *      to radius.
+ * - 5: The 10 blocks that form a second outer shell. This layer is
+ *      spherical, so a logarithmic map can optionally be used in this layer.
+ *      This allows the domain to extend to large radial distances from the
+ *      compact objects. This layer can be h-refined radially, effectively
```

Comment:
Why "effectively"? Is it because they're technically not S2xI1? I think it would be clearer to ignore that subtlety in the documentation for the sake of easier reading